### PR TITLE
[8.0] Add ngo_purchase module

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: ngo_purchase/README.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+NGO - Purchase
+==============
+
+This is a glue module to define a specific view of Purchase Order
+for NGO verticalization. The new view intent to give consistency with
```
Comment (lepistone) [inline ngo_purchase/README.rst]: s/intent/is meant to

or something else

**Comment by lepistone** [comment]
:+1: thanks (minor typo in the description)

**Comment by gurneyalex** [comment]
:+1:

**Comment by yvaucher** [comment]
Fixed typo

File: ngo_purchase/model/purchase_order.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    req_bid_tendering_mode = fields.Selection(
+        [('open', 'Open'),
+         ('restricted', 'Restricted')],
+        'Call for Bids Mode',
+        readonly=True,
+        help="- Restricted : you select yourself the bidders and generate a "
+             "RFQ for each of those. \n- Open : anybody can bid (you have to "
+             "advertise the call for bids) and you directly encode the bids "
+             "you received. You are still able to generate RFQ if you want to "
+             "contact usual bidders.")
```
Comment (gurneyalex) [inline ngo_purchase/model/purchase_order.py]: the help string needs to be rewritten (e.g. "the bid tendering mode of the related purchase requisition")

File: ngo_purchase/model/purchase_order.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+from openerp import models, fields
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    req_bid_tendering_mode = fields.Selection(
```
Comment (gurneyalex) [inline ngo_purchase/model/purchase_order.py:27]: should be a related field

**Comment by yvaucher** [comment]
@gurneyalex thanks

**Pull request merged**